### DISCUSSION TITLE
Certain paths don't show up

### DISCUSSION THREAD

**Discussion by littlemandave** [meerk40t/meerk40t#1657] [General]
I have an inkscape svg file (attached) where some of the glyphs don't show up when I open it in meerk40t (Using 0.8.1 on a Mac). All glyphs are converted to paths in inkscape.

In particular the sharp and flat symbols are invisible in meerk40t except for one flat symbol!?!? I'm flummoxed. The same file opens fine and shows all glyphs in every other program I viewed it in. I'm probably just being dense, but I can't figure it out.

Could some kind, interested soul take a look? Here's the .svg file. Thanks in advance.

![sharpflattest](https://user-images.githubusercontent.com/72283130/212429265-72e335f5-f9fd-44a6-980c-3653744dda53.svg)

**Comment by jpirnay** [comment]
Just reading your comment, I seem to be missing the file you refer to

**Reply by littlemandave** [reply]
I resolved the issue, thanks! But for the record the file is there for me. It’s black glyphs on a transparent bag, maybe you just can’t see it?

**Comment by jpirnay** [comment]
Thanks, I did not recognise the file on the black background

**Comment by littlemandave** [comment]
I’m still slightly flummoxed, but I have a workaround. It seems that many of those glyphs didn’t have their fill color set explicitly to black, instead it was set to “current color“ or something like that. You can see it if you look at the file as XML. For some reason meerk40t is the only app I found that didn’t automatically use a black fill on those glyphs. But I can select them in Inkscape, explicitly set the fill color, and then they show up in meerk40t. On Feb 23, 2023, at 12:41 PM, jpirnay ***@***.***> wrote:﻿
Thanks, I did not recognise the file on the black background

**Comment by jpirnay** [comment]
This is a bug and has been addressed. Thank you.

**Comment by littlemandave** [comment]
﻿Great! Thanks for letting me know. On Feb 24, 2023, at 1:19 AM, jpirnay ***@***.***> wrote:﻿
This is a bug and has been addressed. Thank you.
